WebPages/WebPages/src/js/console.js

```diff
@@ -634,16 +634,16 @@
         return v;
       },
       temp: function(v) {
-        return v.toPrecision(6) + "\u00B0C";
+          return v.toFixed(2) + " K"; 
       },
       pres: function(v) {
-        return siUnit(v / 1000, "Pa");
```

Comment:
the `siUnit()` function should be handling whether to add `k`, `M`, etc, with the passed in string as the base unit (in this case, pascals).  Did you find an issue in that function that prevented it from using the proper units?

Reply:
line 1410 is probably where you want to adjust the `.toPrecision()` values for things that use `siUnit()`

Reply:
All that being said, I don't have a problem with your solution here :)  I'm just thinking that if it's a problem here, where else might it be a problem.  If we can fix it in the function, then the fix propagates to all the consumers instead of just in this spot.

Reply:
Thanks for the remark.
I didn't find a specific problem with siUnit, other than that it does not seem to do the job.
Removing the " / 1000 " results in the correct value, but not the correct unit prefix k
Changing the " / 1000 " to " * 1000" results in correct result: 100234.60 Pa, but it seems that siUnit should compact that.

siUnit() is also called for distance and velocity.
Adjusting to .toFixed(2) in siUnit() will also mean those will change. Specifically distance may get more accyrate results at VERY extreme solar orbits (238520.60 Gm height from terrain with toFixed(2) opposed to 238521 GM with toPrecision(6). )
So, it seems the to.Fixed(2) can be implemented there.

As to why siUnit() works with distance and not with Pressure, I don't know yet.
I'l try to look into that, it eludes me for now.